scripts/daqutils

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+
+# Silent pushd and popd
+pushd () {
+    command pushd "$@" > /dev/null
+}
+
+popd () {
+    command popd "$@" > /dev/null
+}
+
+
+# Go to source directory to make python import works
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+pushd $SCRIPT_DIR
+python3 run_daq_utils.py $@
+popd
```

Comment:
There's no need to `pushd` and `popd` inside a bash script. You can `cd` and the calling shell will always stay in the original location. This would only be needed if this script was being sourced.

Reply:
Changed this to cd.

Reply:
Fixed.